health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/NewCampaignCreate/CreateCampaign.js

```
@@ -0,0 +1,197 @@
+  const { t } = useTranslation();
+  const history = useHistory();
+  const [showToast, setShowToast] = useState(null);
+  const [totalFormData, setTotalFormData] = useState({});
+  const [campaignConfig, setCampaignConfig] = useState(CampaignCreateConfig(totalFormData));
+  const [params, setParams] = Digit.Hooks.useSessionStorage("HCM_ADMIN_CONSOLE_DATA", {});
+  const [loader, setLoader] = useState(null);
+  const [defaultFormValues, setDefaultFormValues] = useState({});
+  const searchParams = new URLSearchParams(location.search);
+  const [currentKey, setCurrentKey] = useState(() => {
+    const keyParam = searchParams.get("key");
+    return keyParam ? parseInt(keyParam) : 1;
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)